src/metatrain/experimental/nanopet/default-hypers.yaml

```diff
@@ -11,6 +11,15 @@ architecture:
     num_gnn_layers: 2
     heads: {}
     zbl: False
+    long_range: 
+      enable: false
+      calculator: ewald
+      prefactor: 1.0
+      atomic_smearing: 1.0
+      lr_wavelength: 0.5
```

Comment:
Set the accuracy using the tuning code - Maybe based on the first structure of training set.